test/server/lib/OIDCConfig.ts

```diff
@@ -241,6 +242,36 @@ describe('OIDCConfig', () => {
         });
       });
     });
+
+  describe('GRIST_HTTPS_PROXY', function () {
+    const proxyURL = 'http://localhost-proxy8080';
```

Comment:
Nitpicking in case you would have liked to specify a port:

```suggestion
    const proxyURL = 'http://localhost-proxy:8080';
```